samples/frameworks/react-native-expo/ts/messaging/src/ehReceiveEvents.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { EventHubConsumerClient, earliestEventPosition, WebSocketImpl } from "@azure/event-hubs";
+import { WebSocketWrapper } from "./wsWrapper";
+
+const connectionString = process.env["EVENTHUB_CONNECTION_STRING"] || "";
+const eventHubName = process.env["EVENTHUB_NAME"] || "";
+const consumerGroup = process.env["CONSUMER_GROUP_NAME"] || "";
+
+export async function main() {
+  console.log(`Running receiveEvents sample`);
+
+  const consumerClient = new EventHubConsumerClient(consumerGroup, connectionString, eventHubName, {
+    webSocketOptions: {
+      webSocket: WebSocketWrapper as WebSocketImpl,
+    },
+  });
+
+  const subscription = consumerClient.subscribe(
+    {
+      processEvents: async (events, context) => {
+        for (const event of events) {
+          console.log(
+            `Received event: '${event.body}' from partition: '${context.partitionId}' and consumer group: '${context.consumerGroup}'`
+          );
+        }
+      },
+      processError: async (err, context) => {
+        console.log(`Error on partition "${context.partitionId}": ${err}`);
+      },
+    },
+    { startPosition: earliestEventPosition }
+  );
+
+  // Wait for a bit before cleaning up the sample
+  setTimeout(async () => {
+    await subscription.close();
+    await consumerClient.close();
+    console.log(`Exiting receiveEvents sample`);
+  }, 30 * 1000);
```

Comment:
is this timeout value reasonable in the react setting?

Reply:
It probably wouldn't matter much if the mobile app keeps running. We keep streaming events for a period of time.  I've shortened it to 10 seconds though.